SamcoEnhanced/SamcoEnhanced.ino

```diff
@@ -368,8 +368,10 @@ bool buttonPressed = false;                      // Sanity check.
     #endif // USES_SOLENOID
     #ifdef USES_DISPLAY
     bool serialDisplayChange = false;                // Signal of pending display update, sent by Core 2 to be used by Core 1 in dual core configs
-    uint8_t serialLifeCount = 0;
+    uint16_t serialLifeCount = 0;//Changed from uint16_t for games with life values > 255
     uint8_t serialAmmoCount = 0;
+    uint16_t VidaMax = 0; //Max value for life in lifebar mode (100%)
+    uint16_t Porcentaje = 0; //Actual value to show in lifebar mode #%
```

Comment:
Me habla no en espanol. Please use English to keep the language consistent.

Reply:
Done!

Reply:
Much better. :)